code/src/App.js

```diff
@@ -1,9 +1,33 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Type } from './Components/Type.js'
+import { Favorit } from './Components/Favorit.js'
```

Comment:
Please type in English instead of Swedish when you code
```suggestion
import { Favorite } from './Components/Favorite.js'
```